src/script/view_model/panel/PanelHeader.tsx

```diff
@@ -29,26 +29,46 @@ export interface PanelHeaderProps {
   onClose: () => void;
   onGoBack: () => void;
   title?: string;
+  tabIndex?: number;
+  handleBlur?: () => void;
 }
 
-const PanelHeader: React.FC<PanelHeaderProps> = ({onGoBack, onClose, title, goBackUie, closeUie = 'do-close'}) => {
+const PanelHeader: React.ForwardRefRenderFunction<HTMLButtonElement, PanelHeaderProps> = (
+  {onGoBack, onClose, title, goBackUie, closeUie = 'do-close', tabIndex = 0, handleBlur}: PanelHeaderProps,
+  ref: React.ForwardedRef<HTMLButtonElement>,
+) => {
   return (
     <div className="panel__header">
       <DragableClickWrapper onClick={onGoBack}>
-        <button className="icon-button" data-uie-name={goBackUie} title={t('index.goBack')}>
+        <button
+          id="arrowleftiid"
+          ref={ref}
+          className="icon-button"
+          data-uie-name={goBackUie}
+          title={t('index.goBack')}
+          tabIndex={tabIndex}
+        >
           <Icon.ArrowLeft />
         </button>
       </DragableClickWrapper>
-
-      <div className="panel__header__title">{title}</div>
-
+      {/*eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex*/}
+      <div className="panel__header__title" tabIndex={0}>
+        {title}
+      </div>
       <DragableClickWrapper onClick={onClose}>
-        <button className="icon-button" data-uie-name={closeUie} title={t('accessibility.rightPanel.close')}>
+        <button
+          className="icon-button"
+          data-uie-name={closeUie}
+          title={t('accessibility.rightPanel.close')}
+          tabIndex={tabIndex}
+          onBlur={handleBlur}
+        >
           <Icon.Close className="right-panel-close" />
         </button>
       </DragableClickWrapper>
     </div>
   );
 };
 
-export default PanelHeader;
+const PanelHeaderForwarded = React.forwardRef(PanelHeader);
+export default PanelHeaderForwarded;
```

Comment:
```suggestion
export default forwardRef(PanelHeader);
```

It should work the same.